test/Microsoft.SqlTools.ServiceLayer.UnitTests/Connection/ConnectionServiceTests.cs

```diff
@@ -247,6 +247,27 @@ public void CanCancelConnectRequestByDisconnecting()
             Assert.False(disconnectResult);
         }
 
+        /// <summary>
+        /// Verify that we cannot connect to the default database when no username is
+        /// provided as a parameter.
+        /// </summary>
+        [Test]
+        public async Task CantConnectWithoutUsername([Values(null, "")]string databaseName)
```

Comment:
You don't appear to actually be using the databaseName parameter. Copy/paste error?